galsim/config/value_eval.py

```diff
@@ -60,7 +60,7 @@ def _type_by_letter(key):
 eval_base_variables = [ 'image_pos', 'world_pos', 'image_center', 'image_origin', 'image_bounds',
                         'image_xsize', 'image_ysize', 'stamp_xsize', 'stamp_ysize', 'pixel_scale',
                         'wcs', 'rng', 'file_num', 'image_num', 'obj_num', 'start_obj_num',
-                        'world_center', 'sky_pos', 'bandpass' ]
+                        'world_center', 'sky_pos', 'uv_pos', 'bandpass' ]
```

Comment:
Is bandpass already in the docs? If so ignore me. Otherwise I don't think I saw it added in this or though I could have missed it.

Reply:
Yes.  It's not in the v2.2 docs, since bandpass wasn't allowed then.  But it was already added to the docs on main when I added that functionality.